website/api/zkvm/quickstart.md

```diff
@@ -5,93 +5,70 @@ slug: ./quickstart
 
 # zkVM Quick Start
 
-Welcome to the [zkVM] Quick Start page! Here are the steps between you and your first proof:
+Welcome to the zkVM Quick Start page! Here are the steps to create your first proof:
 
-1. Install the RISC Zero toolchain.
-2. Initialize a new project using the [cargo risczero] tool.
-3. Edit the boilerplate to build your first project.
-4. Run your project locally or with [Bonsai].
+1. Install the `cargo-risczero` toolchain
+2. Create a new project using the `cargo-risczero` tool.
+3. Familiarize yourself with the project's structure
+4. Run your project locally
+
+## Concept break
+
+The zkVM provides a zero-knowledge proof of the correct execution of Rust-based program.
+The **[host]** is responsible for launching the zkVM (or a [prover]), and then the **[guest]** is the program that runs inside the zkVM.
+A proof of execution is a [receipt]; it contains a public part - a [journal] and an encryption part - a [seal].
 
 ## 1. Install the RISC Zero Toolchain
 
-If you have already installed Rust via `rustup`, you can install our toolchain by running
+First, make sure you [install Rust], then install the RISC Zero toolchain by running the following:
 
 ```bash
 cargo install cargo-binstall
 cargo binstall cargo-risczero
 cargo risczero install
 ```
 
-If you need to install Rust or encounter problems, take a look at our [full installation instructions](install.md).
-
-## 2. Initialize a New Project
+## 2. Create a New Project
 
-Once you've installed the toolchain, you can initialize a new project using the [starter template] by running:
-[starter template]: https://github.com/risc0/risc0/tree/main/templates/rust-starter
+The `cargo-risczero` tool takes `--guest-name` parameter, a [guest] program that the zkVM runs, to generate a proof of its execution:
 
 ```bash
-cargo risczero new my_project
+cargo risczero new my_project --guest-name guest_code_for_zk_proof
 ```
 
-See the [cargo risczero] docs for other options for initialization.
+There is a list of options in the [feature flags]. To create a proof with the zkVM on your own machine, we recommend at least 16 GB of RAM.
```

Comment:
also I don't think the feature flags is a very user-friendly place to land when diving into a quick start guide. 

imo the pointer to the cargo risczero docs makes more sense. (if those docs don't include all the feature flag information, we should improve the cargo docs)

Reply:
This quick start guide is an updated place for zkhack hackers to get all the info they need - we are unlikely going to update cargo docs today